content/diff_models.qmd

```diff
@@ -0,0 +1,147 @@
+
+The resulting 1024 image-tokens can then be fed into the decoder-Block of the VAE to generate an image.
+An illustration of the training-process can be seen in @fig-dVAE.
+
+<!-- ![Illustration of the DALL-E training process.](../imgs/dalle_architecture.png){#fig-dVAE .enlarge-onhover} -->
+
+### CLIP
+
+Close to the release of DALL-E, the team at OpenAI did also publish CLIP [@radfordLearningTransferableVisual2021].
+The paper, which introduced a contrastive^[Contrastive also being the namesake of the method (Contrastive Language-Image Pre-training)] method to learn visual representations from images and text descriptions, bridged the gap between image and text embeddings. This contrastive principle is illustrated in @fig-CLIP.
+
+![Illustration of the contrastive learning paradigm used in CLIP, taken from @radfordLearningTransferableVisual2021](../imgs/clip.png){#fig-CLIP .enlarge-onhover}
+
+A matrix of all combinations of images and text descriptions is created. The model then learns to predict the correct image for a given text description and vice versa. This is done by encoding both the image and the text into a vector space, which is then used to calculate the similarity between the two vectors.
+to do this, both a vision- and a text-transformer are trained as encoders to maximize the cosine similarity between the encoded image and text for each pair in the matrix and minimizing it for all other pairs.
+The authors also show that this method can be used to transfer the learned representations to other tasks, such as zero-shot classification.
+
+## Diffusion Models
+
+Though models like DALL-E and CLIP represented significant milestones in the journey of text-to-image generation, the field continued to evolve rapidly, leading to the advent of Stable Diffusion. This evolution was partly inspired by the need for more control over the generation process and a desire for higher-quality outputs at lower computational costs.
+
+The GAN-architecture (first published in 2014) was the de-facto standard for quite some time and though the central principle of their successors diffusion models was published in 2015 [@sohl-dicksteinDeepUnsupervisedLearning2015], it took until 2020 for them to beat GANs on most benchmarks [@dhariwalDiffusionModelsBeat2021].
+
+The diffusion model's central principle is the training on a sequence of gradually noised images. Here's the updated snippet with the two reasons for step-by-step noise addition included:
```

Comment:
almost word-for-word repetition.

The diffusion model's central principle is the training on a sequence of gradually noised images. Here's the updated snippet with the two reasons for step-by-step noise addition included:

The diffusion model's central principle is training on a sequence of gradually noised images.

Reply:
however did that happen :thinking_face: